masterlist.yaml

```
@@ -16672,3 +16672,15 @@ plugins:
       - <<: *alreadyInX
         subs: [ 'CCOR_Vokrii.esp']
         condition: 'active("CCOR_Vokrii.esp") and active("Vokrii - WACCF Patch.esp")'
+
+  ### Raven Rock Enhanced ###
+  - name: 'RREnhanced.esp'
+    dirty:
```

That's fine I'd add a link just for future updates.
```suggestion
  - name: 'RREnhanced.esp'
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/19001' ]
    dirty:
```